apps/desktop/src/platform/preload.ts

```diff
@@ -71,6 +72,27 @@ const clipboard = {
   write: (message: ClipboardWriteMessage) => ipcRenderer.invoke("clipboard.write", message),
 };
 
+const sshagent = {
+  setKeys: (keys: { name: string; privateKey: string; uuid: string }[]): Promise<void> =>
+    ipcRenderer.invoke("sshagent.setkeys", keys),
+  signRequestResponse: async (id: number, accepted: boolean) => {
+    await ipcRenderer.invoke("sshagent.signrequestresponse", { id: id, accepted: accepted });
+  },
+  generateKey: async (keyAlgorithm: string): Promise<ssh.SshKey> => {
+    return await ipcRenderer.invoke("sshagent.generatekey", { keyAlgorithm: keyAlgorithm });
```

Comment:
suggestion: simplify
```suggestion
    return await ipcRenderer.invoke("sshagent.generatekey", { keyAlgorithm });
```